docs/release-process.md

```diff
@@ -92,6 +92,7 @@ The final release is cut. If an urgent and important issue needs to be addressed
 2. Build:
     - Tag a new release (e.g. 1.1.0) and run an official build which should be essentially identical to the last RC
     - Post in Desktop App channel with links to all supported distributions and SHA-256 checksum
+    - Complete code-signing and confirm that it worked well by testing the download links and keeping an eye out for any warnings or issues
```

Comment:
I also used some commands to verify signatures. It would be good to note them here.

Windows:
On "Developer Command Prompt" (bundled in Visual Studio),
```bat
signtool verify /pa /all EXE_TO_VERIFY
```

Mac:
On console,
```bash
codesign --verify --deep --strict --verbose=2 Mattermost.app
```